src/logging.rs

```diff
@@ -345,3 +609,36 @@ fn on_close_button(
         }
     }
 }
+
+fn on_clear_button(
```

Comment:
lets have a button enum and only a single `on_button` system that does the whole `Interaction` shizzle